scripts/mancheck.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN
+# AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT
+# OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+# shellcheck disable=SC2086
+
```

Comment:
Let's throw in some basic usage validation to avoid this sort of thing:
```
FreeBSD-13_0-ALPHA2-02611ef8ee9 ➜  zfs git:(manlint) ./scripts/mancheck.sh
find: illegal option -- t
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
load: 0.29  cmd: awk 35191 [ttyin] 7.12r 0.00u 0.00s 0% 2800k
mi_switch+0xc1 sleepq_catch_signals+0x2e6 sleepq_wait_sig+0x9 _cv_wait_sig+0xe4 tty_wait+0x1c ttydisc_read+0x2ac ttydev_read+0x56 devfs_read_f+0xd5 dofileread+0x81 sys_read+0xbc amd64_syscall+0x10c fast_syscall_common+0xf8


^C
```

Reply:
Added a usage thing when called with no arguments and an exit if the find fails to prevent blocking on stdin